cluster/skeleton/util.sh

```diff
@@ -28,11 +28,6 @@ function detect-node-names {
 	echo "NODE_NAMES: [${NODE_NAMES[*]}]" 1>&2
 }
 
-# Get node IP addresses and store in KUBE_NODE_IP_ADDRESSES[]
```

Comment:
Why deleting this from skeleton?

Reply:
See other comment - there are no callers of detect-nodes() in the entire tree.

Reply:
I'd prefer sending a separate cleanup PR later.